src/EnergyPlus/HeatBalanceSurfaceManager.cc

```diff
@@ -3558,19 +3557,20 @@ void InitIntSolarDistribution(EnergyPlusData &state)
 
         for (int enclosureNum = 1; enclosureNum <= state.dataViewFactor->NumOfSolarEnclosures; ++enclosureNum) {
 
-            if (state.dataHeatBalSurf->RecDifShortFromZ(enclosureNum)) {
+            if (state.dataHeatBalSurf->ZoneRecDifShortFromZ(enclosureNum)) {
```

Comment:
So, here it seems clear that `ZoneRecDifShortFromZ` is an enclosure variable, I think?